CONTRIBUTING.md

```diff
@@ -1,3 +1,31 @@
+# Contributing to ROS 2 Java
+
+This document contains rules and guidelines for contributing to the ROS 2 Java project.
+
+## How to contribute
+
+Here are a few things you can do to contribute to the project:
+
+1. Report a bug by opening an issue on GitHub
+2. Even better, open a **pull request** fixing the bug!
```

Comment:
We could have issue and pull request templates in the future

Reply:
Yes, I think that's a good idea.